docs/UserGuide.md

```diff
@@ -374,16 +441,32 @@ Format: `clear`
 [Return to table of contents](#table-of-contents)
 
 
-### Closing the app: `exit`
+### Closing the app : `exit`
 
 Exits the application.
 
 Format: `exit`
 
 [Return to table of contents](#table-of-contents)
 
+### Viewing help : `help`
+
+Shows a message explaining how to access the help page.
+
+![help message](images/helpMessage.png)
+
+Format: `help`
+
+[Return to table of contents](#table-of-contents)
+
+### Accessing command history
+
+Using the up and down arrow keys on the keyboard retrieves past command lines entered. 
```

Comment:
Should it be `Uses...` instead to be consistent with other sections?

```suggestion
Uses the up and down arrow keys on the keyboard retrieves past command lines entered. 
```

Reply:
hmm compared to:
"Retrieves past command lines entered using the up and down arrow keys. "
i'm more inclined towards the original
"Using the up and down arrow keys on the keyboard retrieves past command lines entered. "

Reply:
Hmm how about "Scrolls through the command history using the up and down arrow keys"? I do agree that the original sounds smoother, but since the descriptions of other sections always begin with a verb, I think it would be nice if we can follow that convention

![image](https://user-images.githubusercontent.com/63232300/139905267-be561494-90c9-44c4-9983-5855bc95376b.png)